pages/WhatIsOppia.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities ("explorations") which try to recreate the
```

Comment:
Or should we say "...share **explorations** (interactive activities) which try to recreate ...", so the term **explorations** attract more attention from readers?